scripts/upgrade.sh.in

```diff
@@ -75,6 +75,8 @@ upgrade_bootstrap () {
         metalk8s:endpoints:salt-master --out txt | cut -d' ' -f2- )"
     repo_endpoint="$($SALT_CALL pillar.get \
         metalk8s:endpoints:repositories --out txt | cut -d' ' -f2- )"
+    certificates=$($SALT_CALL pillar.get \
+        certificates --out txt | cut -d' ' -f2-)
```

Comment:
We namespace all other pillar values. Shouldn't this as well?

Reply:
We're not namespacing everything (e.g. networks, proxies, signing policies), still we could.

Reply:
If we want to, warrants another ticket.